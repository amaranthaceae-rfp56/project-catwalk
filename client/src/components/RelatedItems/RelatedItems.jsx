import React from 'react';


const RelatedItems = () => {
  return (
    <div data-testid = {'Related-Items'}>
      RELATED ITEMS
    </div>
  );
};

export default RelatedItems;