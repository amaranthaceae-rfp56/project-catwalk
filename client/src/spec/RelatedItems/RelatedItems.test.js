import {render, screen, cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import React from 'react';

import RelatedItems from '../../components/RelatedItems/RelatedItems.jsx';

afterEach( ()=> {
  cleanup();
});

test('test', () => {
  render(<RelatedItems/>);
  const item = screen.getByTestId('Related-Items');

  expect(item).toBeInTheDocument();
  expect(item).toHaveTextContent('RELATED ITEMS');
});