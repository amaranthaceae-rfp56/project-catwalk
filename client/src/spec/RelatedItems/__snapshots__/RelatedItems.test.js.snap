// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots 1`] = `
<div
  data-testid="Related-Items"
>
  RELATED ITEMS
</div>
`;
